doc/man1/openssl-verify.pod.in

```diff
@@ -336,45 +337,45 @@ determined.
+The chain is built up step by step, looking up the issuer's certificate of the
+current certificate. If a certificate is found that appears to be its own issuer
+it is assumed to be the self-signed root, which must be trusted.
+
+The process of 'looking up the issuer's certificate' itself involves a number of
+steps. All available certificates with a subject name that matches the issuer
+name of the current certificate are subject to further tests.
 The relevant authority key identifier components of the current certificate (if
 present) must match the subject key identifier (if present) and issuer and
-serial number of the candidate issuer, in addition the keyUsage extension of
-the candidate issuer (if present) must permit certificate signing.
+serial number of the candidate issuer certificate.
 
-The lookup first looks in the list of untrusted certificates and if no match
-is found the remaining lookups are from the trusted certificates. The root CA
-is always looked up in the trusted certificate list: if the certificate to
-verify is a root certificate then an exact match must be found in the trusted
-list.
+The lookup first uses the list of non-trusted "intermediate" CA certificates and
+if no match is found the remaining lookups are from the trusted certificates.
+The root CA is always looked up in the trusted certificate list:
+if the certificate to verify is a self-signed certificate
+then an exact match must be found in the trust store.
 
 The second operation is to check every untrusted certificate's extensions for
 consistency with the supplied purpose. If the B<-purpose> option is not included
-then no checks are done. The supplied or "leaf" certificate must have extensions
+then no checks are done. The target or "leaf" certificate must have extensions
 compatible with the supplied purpose and all other certificates must also be
 valid CA certificates. The precise extensions required are described in more
 detail in L<openssl-x509(1)/CERTIFICATE EXTENSIONS>.
 
-The third operation is to check the trust settings on the root CA. The root CA
-should be trusted for the supplied purpose.
+The third operation is to check the trust settings on the root CA certificate.
+The root CA should be trusted for the supplied purpose.
 For compatibility with previous versions of OpenSSL, a certificate with no
 trust settings is considered to be valid for all purposes.
 
 The final operation is to check the validity of the certificate chain. The
 validity period is checked against the current system time and the notBefore
 and notAfter dates in the certificate. The certificate signatures are also
-checked at this point.
+checked (except for the signature of the self-signed root certificate,
+which is verified only if the B<-check_ss_sig> option is given). When verifying
+a certificate signature the keyUsage extension (if present) of the candidate
+issuer certificate is checked to permit the respective signing operation.
```

Comment:
This last sentence is a bit strange : 
> When verifying a certificate signature the keyUsage extension (if present) of the candidate
> issuer certificate is **checked to permit the respective signing operation.**

You say that the code is checking "that each candidate issuer has the right keyUsage for signing ?"
i.;e was able to sign respectively to its keyUsage  extension ?

Reply:
Good that you ask - the "respective" was too vague/abstract. 
I've expanded the sentence to
```
When verifying
a certificate signature the keyUsage extension (if present) of the candidate
issuer certificate is checked to permit the signing operation: digitalSignature
for signing proxy certifiates and keyCertSign for signing other certificates.
```